### PR TITLE
Multiprotocol working on sky9x

### PR DESCRIPTION
**Pull request by kilrah**
Telemetry not working yet though, still something to debug

### CODE REVIEW

File: radio/src/pulses/multi_arm.cpp [hunk 1]
```diff
@@ -46,7 +46,7 @@ static void sendByteMulti(uint8_t b)
     parity = parity ^ (b & 1);
     b >>= 1;
   }
-  putDsm2SerialBit(parity);      // Even Parity bit
+  putDsm2SerialBit(!parity);     // Even Parity bit
```
Comment (schwabe) [inline radio/src/pulses/multi_arm.cpp]: I am not sure about this change. Wouldn't that change the parity on the other platforms as well?

Comment (kilrah) [inline radio/src/pulses/multi_arm.cpp]: It's only for PPM_PIN_SERIAL, which I believe isn't used on any other platforms, which explains it was never noticed that it was wrong.

Comment (schwabe) [inline radio/src/pulses/multi_arm.cpp]: Oh then did not see enough context. Thought horus was that too, never mind then.

**Comment by bsongis** [comment]
Perhaps you could test with my latest PR?

**Comment by kilrah** [comment]
Nope doesn't help unfortunately

**Pull request merged**